src/Novell.Directory.Ldap.NETStandard/LdapControl.cs

```diff
@@ -1,211 +1,209 @@
-/******************************************************************************
-* The MIT License
```

Comment:
I think you reformatted this file - as everything appears as removed & added, so I can not really see the changes. Can you please fix this.

Reply:
Sorry, the file had one line that ended in CRLF and the rest ended in LF and Visual Studio said the file didn't have consistent file endings and converted them all to CRLF. I reverted the line endings.